### ISSUE TITLE
Parquet format outputs binary array

### ISSUE THREAD

**Issue by furkankahvecinsider** [ClickHouse/ClickHouse#44835]
We're seeing binary array in the parquet file when we're gonna output parquet file to S3 bucket.

My used query

```
SELECT
    name,
    surname,
    age
FROM my_table
INTO OUTFILE 'my_s3_bucket_file_path.pq'
FORMAT Parquet
```

Example for binary array

`{"name":{"type":"Buffer","data":[50,54,49,50,50,49,56,57,57,55,52,48,50,49,98,50,98,49,97,101,50,99,54,46,99,57,48,100,57,98,50,46]}}`

We are using 22.3.3 version of ClickHouse.

What can we do in this case? According to my research, we need to adjust the [output_format_parquet_string_as_string](https://clickhouse.com/docs/en/operations/settings/settings/#output_format_parquet_string_as_string) config, but this config is avaliable in [22.6.1](https://github.com/ClickHouse/ClickHouse/blob/a404b85dc7656c466bd5f23f4168fb358b5bfe23/docs/changelogs/v22.6.1.1985-stable.md) version. Do we need to upgrade to that version?

------

By the way, we are using version 20.12.8 of ClickHouse on another different machine and when we run the above query, we do not have the above problem. We don't know what differences there might be between version 20.12.8 and version 22.3.3.

**Comment by Avogar** [comment]
Before version 21.7 we used Parquet type `string` to output ClickHouse `String` type. But it's totally incorrect, because ClickHouse `String` type can contain arbitrary binary data and invalud UTF8 sequences. In 21.7 we changed Parquet type from `string` to `binary`. 
The setting `output_format_parquet_string_as_string` was added only in 22.6. Unfortunately, there is no way to use Parquet `string` type instead of `binary` in versions 21.7-22.6. I can backport it to 22.3, but I think the best solution will be using more recent ClickHouse versions.
